src/main/java/com/mathworks/ci/RunMatlabCommandBuilder.java

```diff
@@ -114,10 +124,11 @@ private int execMatlabCommand(FilePath workspace, Launcher launcher,
         // Create MATLAB script
         createMatlabScriptByName(uniqeTmpFolderPath, uniqueCommandFile, workspace, listener, envVars);
         ProcStarter matlabLauncher;
+        String options = getStartupOptions() == null ? "" : getStartupOptions().getOptions();
 
         try {
             matlabLauncher = getProcessToRunMatlabCommand(workspace, launcher, listener, envVars,
-                    "cd('"+ uniqeTmpFolderPath.getRemote().replaceAll("'", "''") +"');"+ uniqueCommandFile, uniqueTmpFldrName);
+                    "cd('"+ uniqeTmpFolderPath.getRemote().replaceAll("'", "''") +"');"+ uniqueCommandFile, options, uniqueTmpFldrName);

```

Comment:
just realized that uniqe (unique) is spelled wrong here lol. Not sure if this mistake is all over or how hard it is to fix